saleor/graphql/attribute/utils.py

```diff
@@ -234,10 +236,18 @@ def clean_input(
         return cleaned_input
 
     @staticmethod
-    def _clean_file_url(file_url: Optional[str]):
+    def _clean_file_url(file_url: Optional[str], error_class):
         # extract storage path from file URL
+        storage_root_url = build_absolute_uri(default_storage.base_url)
+        if file_url is not None and not file_url.startswith(
+            storage_root_url  # type: ignore
+        ):
+            raise ValidationError(
+                "The file_url must be the path to the default storage.",
+                code=error_class.INVALID.value,
+            )
         return (
-            re.sub(f"^{settings.MEDIA_URL}", "", urlparse(file_url).path)
+            re.sub(f"^{default_storage.base_url}", "", urlparse(file_url).path)
             if file_url is not None
```

Comment:
this `if` doesn't make sense, as few lines above you `raise ValidationError` in such situation.

Reply:
No, I'm not raising an error for such a situation.